main.py

```diff
@@ -0,0 +1,53 @@
+import math
+import csv
+import requests
+from bs4 import BeautifulSoup
+import sys
+
+
+def get_distance(loc1, loc2):
+    return round(math.sqrt((loc1[0] - loc2[0]) * (loc1[0] - loc2[0]) + (loc1[1] - loc2[1]) * (loc1[1] - loc2[1])), 4)
+
+
+def bye(message):
+    print(message)
+    quit()
+
+
+#  extra idea for the future - use this to get your location live
+
+# send_url = "http://api.ipstack.com/check?access_key=da96285c376c412b6bab56d3dfaa4961"
```

Comment:
Usually, one doesn't want to add api keys in plain text into svc, more so on public repositories.

What could happen: https://www.theregister.com/2015/01/06/dev_blunder_shows_github_crawling_with_keyslurping_bots/